includes/event/event-repository-interface.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Wporg\TranslationEvents\Event;
+
+use Exception;
+use Throwable;
+use WP_Error;
+
+class EventNotFound extends Exception {
```

Comment:
If you execute the PHPCS with the WordPress standard for this file

`phpcs includes/event/event-repository-interface.php --standard=WordPress`

you will see some errors, like each class should be in its own file. I know it is some extreme to split these small classes in different files, but this is what the WPCS suggests. You can see [here](https://github.com/WordPress/WordPress-Coding-Standards/pull/1802) the PR for this:

> Best practice suggestion: Declare only one class/interface/trait in a file.

Reply:
There are quite a few [rules that we are disabling](https://github.com/WordPress/wporg-gp-translation-events/blob/trunk/phpcs.xml#L15-L25). I would prefer to not make changes to phpcs in this repo, as that will require changing code that is not introduced by this PR, and in this PR the intention was just to add the repository without making any changes to the existing code.

If we want to change the phpcs rules, my suggestion would be to do that in a PR that only does that, as it will likely result in many changes across the codebase.